### DISCUSSION TITLE
Plotman Config Question

### DISCUSSION THREAD

**Discussion by Jericon** [ericaltendorf/plotman#829] [Q&A]
Hello,

I have a Multi-Server setup.  One main farmer that also does plotting, a dedicated plotting machine and my old farmer (which is now just a harvester).

I am trying to figure out the optimal configuration.  On the Plotter, I want to use rsyncd for archiving, but the destination on my farming server is a large directory.  There are not multiple mount points for each drive.  The server has more than 10 Drives with hardware raid to configure them.  The Guides and Wiki are not clear how to set this up.

Likewise, the plotter is using a 110 GB RAM disk as -2.  It does not have sufficient ram to increase that.  Using the Mad Max Plotter, I'd like to try to get multiple plots going simultaneously, but limit to one plot using -2 at a time.  The documentation is not clear how to configure that in the scheduler.

Should note that all of the hosts are on a private 1GBit Network, with no bandwidth limitations.

Thanks.

**Comment by altendky** [comment]
RAID for farming drives is generally not considered to be good.  One way you get the overhead of redundancy and the lost capacity.  The other you get the hazards of one failing drive taking out more than one drive's worth of data.

You could make your own target config in your `plotman.yaml` similar to below https://github.com/ericaltendorf/plotman/blob/3edef7ef97e0d4670322ae586823f4bbb1fc3c76/src/plotman/resources/target_definitions.yaml but just handling a single directory.  See https://github.com/ericaltendorf/plotman/wiki/Archiving#custom-archiving-setups.  At some point I would like to make a target that just takes a list of directories, but I haven't gotten to that yet.

I just switched to madMAx today.  I have 192GB RAM and was hoping that maybe I could get away with staggering a pair of processes and maybe peak at 175GB or less...  but I did a quick log of usage and while I might be able to play some games for my needs, tmp2 is definitely significantly used throughout the entire process.  On the upside, I got only about a 4% increase in times with no RAM disk at all.  Running two staggered plots more than made up for that.  Or something, I forget the exact numbers...

(the two curves are the same, just time shifted to help look at what the usage might be)
![image](https://user-images.githubusercontent.com/543719/125017304-da4de980-e040-11eb-9c05-205a161d6f54.png)
